src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -52,6 +52,11 @@ internal partial class DataFlowPass : AbstractFlowPass<DataFlowPass.LocalState>
         private readonly PooledHashSet<LocalSymbol> _usedVariables = PooledHashSet<LocalSymbol>.GetInstance();
 
         /// <summary>
+        /// The inferred nullability at the point of declaration of var locals.
+        /// </summary>
+        private readonly PooledDictionary<LocalSymbol, bool?> _variableIsNullable = PooledDictionary<LocalSymbol, bool?>.GetInstance();
```

Comment:
>private readonly PooledDictionary<LocalSymbol, bool?> _variableIsNullable = PooledDictionary<LocalSymbol, bool?>.GetInstance(); [](start = 8, length = 127)

It still doesn't look like this is part of the state and participate in merging.

Reply:
Added `// PROTOTYPE` comment.